content/en/user-guide/chaos-engineering/fault-injection-service/index.md

```diff
@@ -1,50 +1,49 @@
 ---
-title: "Using the Fault Injection Service"
-linkTitle: "Using the Fault Injection Service"
-weight: 1
-description: Use LocalStack Outages Extension to mimic service outages by testing your infrastructure's ability to deploy robustly and recover from unexpected events.
+title: "Fault Injection Service"
+linkTitle: "Fault Injection Service"
+description: Use Fault Injection Service to simulate faults in your infrastructure and test its fault tolerance
```

Comment:
make it a bit more specific: 'Use Fault Injection Service to simulate faults in your infrastructure and test its fault tolerance' is the general description for our chaos features, so this should be more about AWS provided FIS functionality.